src/components/common/SearchInput/SearchInput.tsx

```diff
@@ -35,31 +59,97 @@ const doSearch = async (term: string) => {
   navigate(`${pathname}${search}`)
 }
 
+// const SUGGESTED_PRODUCTS = [
```

Comment:
👀

Reply:
Yeah, I'll remove it, but not now, because I need test all this with real data, this mock is different of api returns